### PR TITLE
Added recipe for biobaseHTTPtools

### PR DESCRIPTION
**Pull request by eggzilla**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eggzilla** [comment]
Please do not merge yet, this is a test for a new haskell deployment. Since GHC requires glibc >2_14 since version 8.4.3 even for its own compilation and conda-forge only provides 2_12 these executables have been statically linked in a alpine linux container with muslc, therefore avoiding the glibc dependency.
As far as I am aware that will be the only possiblity to deploy Haskell programs requiring a newer compiler via bioconda.

**Comment by eggzilla** [comment]
@bgruening after your review i would ping the haskell list and ask for their opinion

File: recipes/biobasehttptools/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+mv AccessionToTaxId ${PREFIX}/bin/AccessionToTaxId
+mv ${PREFIX}/bin/AccessionToTaxId ${PREFIX}/bin/AccessionToTaxId-bin
+echo -e "#!/bin/bash\nexport SYSTEM_CERTIFICATE_PATH=${PREFIX}/ssl/cacert.pem\n${PREFIX}/bin/AccessionToTaxId-bin \"\$@\"\n" > ${PREFIX}/bin/AccessionToTaxId
+chmod 755 ${PREFIX}/bin/AccessionToTaxId
+mv FetchSequence ${PREFIX}/bin/FetchSequence
+mv ${PREFIX}/bin/FetchSequence ${PREFIX}/bin/FetchSequence-bin
+echo -e "#!/bin/bash\nexport SYSTEM_CERTIFICATE_PATH=${PREFIX}/ssl/cacert.pem\n${PREFIX}/bin/FetchSequence-bin \"\$@\"\n" > ${PREFIX}/bin/FetchSequence
+chmod 755 ${PREFIX}/bin/FetchSequence
+mv GeneIdToGOTerms ${PREFIX}/bin/GeneIdToGOTerms
```
Comment (bgruening) [inline recipes/biobasehttptools/build.sh]: `mv GeneIdToGOTerms ${PREFIX}/bin/`

You could also put this into a bash for loop and iterate over all your binaries.

Comment (eggzilla) [reply recipes/biobasehttptools/build.sh]: Nice idea, I will add that right away

Comment (eggzilla) [reply recipes/biobasehttptools/build.sh]: Ok, loop is added

**Comment by epruesse** [comment]
@BiocondaBot

**Comment by epruesse** [comment]
@Biocondabot merge

**Comment by epruesse** [comment]
You know, `for t in tool1 tool2 tool3; do echo $t; done` suffices for this kind of loop.

**Pull request merged**